lib/ansible/modules/cloud/cloudstack/cs_network_offering.py

```diff
@@ -55,10 +62,11 @@
     description:
       - Network offering details in key/value pairs.
       - with service provider as a value
-    choices: [ internallbprovider, publiclbprovider ]
```

Comment:
same here, fix the arg_spec with adding this choices?

Reply:
Here, we are waiting for a list of dict, I think this choice list makes no sense.